server/Models/Tag.cs

```diff
@@ -62,10 +62,11 @@ public bool TypeIsValid(object obj)
         
         public class FTime : IFieldType
         {
+            private static readonly DateTime _epoch = new DateTime(1, 1, 1);
```

Comment:
nit: I have your comment on prev commit about code convention here, however without reading that there is no docs describing this convention (comments or mentioning in readme file might be good ideas)